fasta/fasta-fast.jl

```diff
@@ -3,14 +3,7 @@
 
```

Comment:
Should the changes to this file be cherry-picked out of this branch as an improvement to the existing Julia program?